lib/active_model/serializer/reflection.rb

```diff
@@ -38,6 +38,7 @@ def initialize(*)
         super
         @_links = {}
         @_include_data = true
+        @_meta = nil
```

Comment:
Fix related to ruby warnings. Not sure how these fixes got into this commit :confused: